src/App.tsx

```diff
@@ -1,16 +1,26 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [initialMovies, setInitialMovies]
+  = useState<Movie[]>(moviesFromServer);
```

Comment:
```suggestion
  const [initialMovies, setInitialMovies] = useState<Movie[]>(moviesFromServer);
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/c1ba11f3-f3ab-4f33-99ad-ff70f90731e0)